public/style.css

```diff
@@ -0,0 +1,35 @@
+* {
```

Comment:
I'll have to keep in mind that even with styled components for modules, the proxy will likely need a true CSS.